From: Hannes Reinecke <hare@suse.de>
Date: Fri, 5 Aug 2016 14:41:37 +0200
Subject: libfc: do not send ABTS when resetting exchanges
References: bsc#962846
Patch-Mainline: submitted to linux-scsi, 2016/08/18

When all exchanges are reset the upper layers have already logged
out of the remote port, so the exchanges can be reset without
sending any ABTS.

Signed-off-by: Hannes Reinecke <hare@suse.com>
---
 drivers/scsi/libfc/fc_exch.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/scsi/libfc/fc_exch.c b/drivers/scsi/libfc/fc_exch.c
index 3e9275f..823cc31 100644
--- a/drivers/scsi/libfc/fc_exch.c
+++ b/drivers/scsi/libfc/fc_exch.c
@@ -1860,7 +1860,6 @@ static void fc_exch_reset(struct fc_exch *ep)
 	int rc = 1;
 
 	spin_lock_bh(&ep->ex_lock);
-	fc_exch_abort_locked(ep, 0);
 	ep->state |= FC_EX_RST_CLEANUP;
 	fc_exch_timer_cancel(ep);
 	if (ep->esb_stat & ESB_ST_REC_QUAL)
-- 
1.8.5.6

